client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -37,6 +37,16 @@ const LowerJaw = ({
   const { t } = useTranslation();
   const submitButtonRef = React.createRef<HTMLButtonElement>();
   const testFeedbackRef = React.createRef<HTMLDivElement>();
+  // Wrapper for changing "test-status" only when test code is pressed
+  const [challengeIsCompletedWrapper, setchallengeIsCompletedWrapper] =
+    useState(false);
+
+  const tryToExecuteChallengeWrapper = () => {
```

Comment:
I am not sure if we would need this.

So we could basically look for changes to `challengeIsCompleted` prop in a useEffect. Once it is true, we can persist its value in a state and called it `challengeHasBeenCompleted` or something more descriptive. 

The render the related elements accordingly.